src/components/CallView/shared/VideoVue.vue

```diff
@@ -331,11 +331,11 @@ export default {
 		},
 
 		sessionHash() {
-			return Hex.stringify(SHA1(this.peerId))
+			return Hex.stringify(SHA1(this.nextcloudSessionId))
 		},
 
 		peerData() {
-			let peerData = this.$store.getters.getPeer(this.$store.getters.getToken(), this.peerId, this.model.attributes.userId)
+			let peerData = this.$store.getters.getPeer(this.$store.getters.getToken(), this.nextcloudSessionId, this.model.attributes.userId)
```

Comment:
Just to check, does this method returns nextcloud session id?
https://github.com/nextcloud/spreed/blob/2a38a527d24b26a1c6bbcfc7a752061d54ab7276/lib/Controller/CallController.php#L100

Reply:
Yes, all (unless I am missing something) API endpoints return the Nextcloud session ID, they are not aware of the signaling session ID. Even [the messages sent from the Nextcloud server to the external signaling server](https://github.com/nextcloud/spreed/blob/cc648fd2ebcc26b604bc7e235694dd6aac57c232/lib/Signaling/BackendNotifier.php) include only the Nextcloud session ID, which is then [internally converted to the signaling session ID by the signaling server](https://github.com/strukturag/nextcloud-spreed-signaling/blob/2a1fd2e01875babf3cad299c2ae860aed0f72a2d/backend_server.go#L424).